### PR TITLE
update apparmor 0305

### PR DESCRIPTION
**Pull request by nilxam**

### CODE REVIEW

**Comment by coolo** [comment]
but on the bright side we can cleanup the worker profile too - and possibly we should. It should be limited to read from /var/lib/openqa/share and it should actually be a different user

**Comment by nilxam** [comment]
I'll take a look worker profile.

File: profiles/apparmor.d/usr.share.openqa.script.openqa [hunk 1]
```diff
@@ -19,8 +19,7 @@
   /etc/openqa/database.ini r,
   /etc/openqa/openqa.ini r,
   /proc/meminfo r,
-  /tmp/mojo.tmp rw,
-  /tmp/mojo.tmp.* rw,
+  /tmp/** rw,
   /usr/bin/optipng rix,
```
Comment (coolo) [inline profiles/apparmor.d/usr.share.openqa.script.openqa]: if this is not what we want, I can also give the mktemp a template so we can limit it to /tmp/openqa.* - would that make sense?

Comment (nilxam) [inline profiles/apparmor.d/usr.share.openqa.script.openqa]: I think so, that can avoid the potential risks.

Comment (coolo) [inline profiles/apparmor.d/usr.share.openqa.script.openqa]: /var/lib/openqa/workers is another option - it doesn't have to be a tmp directory actually. I suggest we don't touch anything at all (beside the worker profile) and put an issue in backlog to do that properly

**Comment by coolo** [comment]
https://github.com/os-autoinst/openQA/pull/229 is a followup

**Pull request merged**